<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>desktopActivitySystem</class>
 <widget class="QWidget" name="desktopActivitySystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>desktopActivitySystem</string>
  </property>
  <widget class="QPushButton" name="pushButton_procs">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择进程</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>261</width>
     <height>106</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="proc_name_show">
      <property name="text">
       <string>N/A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="proc_path_show">
      <property name="text">
       <string>N/A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="proc_start_time_show">
      <property name="text">
       <string>N/A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="proc_end_time_show">
      <property name="text">
       <string>N/A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_focus_time_show">
      <property name="text">
       <string>N/A</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>86</width>
     <height>106</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="proc_name_label">
      <property name="text">
       <string>进程名：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="proc_path_label">
      <property name="text">
       <string>路径：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="start_time_label">
      <property name="text">
       <string>启动时间：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="end_time_label">
      <property name="text">
       <string>结束时间：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="focus_time_label">
      <property name="text">
       <string>窗口停留时长：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_show">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
